### ISSUE TITLE
traffic page view 30 days

### ISSUE THREAD

**Issue by npuzl** [FeatureProbe/FeatureProbe#290]
**Is your feature request related to a problem? Please describe.**
On the traffic page, there are only two time intervals: the last 24 hours and the last 7 days. But some switches require longer observation times, such as 30 days. Can it support longer observation time intervals?
![image](https://user-images.githubusercontent.com/53386473/234524031-085f7449-5298-48c3-8af2-7b6b086510fe.png)

**Describe the solution you'd like**
support longger observation time intervals

**Describe alternatives you've considered**

**Additional context**

**Comment by SSebo** [comment]
Yes, 7 days maybe not enough. We'll consider a proper way to support custom range.
